src/muya/lib/parser/marked/lexer.js

```diff
@@ -255,21 +255,41 @@ Lexer.prototype.token = function (src, top) {
         })
 
         // Changing the bullet or ordered list delimiter starts a new list (CommonMark 264 and 265)
+        // Changing to/from task list item from/to bullet, starts a new lit(work for marktext)
```

Comment:
Typo (`lit`, space before `(`) and I would add this line as separate comment because it's unofficial markdown, link to your issue and write why we need this workaround. Maybe we can improve our implementation later to follow the official specification.